common/src/main/java/org/opensearch/ml/common/connector/ConnectorClientConfig.java

```diff
@@ -107,15 +164,34 @@ public static ConnectorClientConfig parse(XContentParser parser) throws IOExcept
                 case READ_TIMEOUT_FIELD:
                     readTimeout = parser.intValue();
                     break;
+                case RETRY_BACKOFF_MILLIS_FIELD:
+                    retryBackoffMillis = parser.intValue();
+                    break;
+                case RETRY_TIMEOUT_SECONDS_FIELD:
+                    retryTimeoutSeconds = parser.intValue();
+                    break;
+                case MAX_RETRY_TIMES_FIELD:
+                    maxRetryTimes = parser.intValue();
+                    break;
+                case RETRY_BACKOFF_POLICY_FIELD:
+                    retryBackoffPolicy = parser.text();
```

Comment:
Add break here to reduce the potential risk.

Reply:
ack